ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py

```diff
@@ -637,7 +606,7 @@ def is_subset_already_added(key, valid_subset_list):
     return False
 
 
-def handle_all_subset(info_subset, valid_subset_list, subset_options, detail):
+def handle_all_subset(info_subset, valid_subset_list, subset_options):
```

Comment:
we are not using detail flag anymore. this should have ben removed long back when i first introduced this function.. just a code cleanup